### ISSUE TITLE
Support for Sec-CH-Prefers-Color-Scheme header

### ISSUE THREAD

**Issue by czM1K3** [brave/brave-browser#39977]
### Platforms

all

### Description

I wanted to use `Sec-CH-Prefers-Color-Scheme` header, but I found out, that Brave doesn't support that although Chromium has been supporting it for ages. It would be nice to have this feature.
Here are some links: [Mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-CH-Prefers-Color-Scheme) and [playground site](https://sec-ch-prefers-color-scheme.glitch.me/).

**Comment by Curstantine** [comment]
Yeah, it would be pretty nice to see this supported. No more flashing sites >_>
